pyproject.toml

```diff
@@ -33,7 +33,7 @@ test = [
 ]
 
 [tool.pdm.build]
-includes = ["dbt_autofix"]
```

Comment:
I'm not familiar with packaging hooks in repos.
Would you have any docs mentioning why we need to add `pre_commit_hooks`?